### PR TITLE
fix for deprecated Symfony Event class

### PR DESCRIPTION
**Pull request by seth-shaw-asu**
**GitHub Issue**: #927 

# What does this Pull Request do?

Updates deprecated Symfony Event class with the new Drupal one and updates the argument order for the dispatch function as specified in https://www.drupal.org/node/3159012.

# What's new?

* Does this change add any new dependencies? No, although it now uses the JWT module's 1.1 release.
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? No.
* Could this change impact execution of existing code? No.

# How should this be tested?

## Short version

Smoke test. Does anything break?

## Longer version

1. `composer require webprofiler`
2. `drush en -y webprofiler`
3. `drush cr` = 💥
4. Apply PR
5. `drush cr` = 👍 (unless you have other modules enabled with the same problem)

# Documentation Status

* Does this change existing behaviour that's currently documented? No.
* Does this change require new pages or sections of documentation? No.
* Who does this need to be documented for? No one.

# Interested parties
@Islandora/committers

### CODE REVIEW

**Review by jordandukart — APPROVED** [review]
Works as described, messages to go out to ActiveMQ and things are generated. Don't see a blow up with webprofiler either.
